test/mcs/test_defmcs.jl

```diff
@@ -11,7 +11,6 @@ using Test
 using Mimi: reset_compdefs, modelinstance, compinstance, 
             get_var_value, OUTER, INNER, ReshapedDistribution
 
-reset_compdefs()
```

Comment:
Why is this removed? (I'm not sure I understand why it was there in the first place, but just wondering what's different now)

Reply:
This follows from the removal of the global list of component defs. Nothing to reset!